README.md

```diff
@@ -104,8 +104,8 @@ Construct a client instance by passing it your Okta domain name and API token:
 [//]: # (method: createClient)
 ```java
 Client client = Clients.builder()
-    .setOrgUrl("{yourOktaDomain}")
-    .setClientCredentials(new TokenClientCredentials("{apiToken}"))
```

Comment:
I feel having these brackets is confusing for some users, who tend to include them while adding values. I've seen users ask questions about it on devforum.